common/components/componentsBySection/CreateEvent/EventOverviewForm.jsx

```diff
@@ -104,14 +104,15 @@ class EventOverviewForm extends React.PureComponent<Props,State> {
                                   currentImage={this.state.formFields.event_thumbnail}
                                   onSelection={this.form.onSelection.bind(this, "event_thumbnail")}
           />
+          <p><em>For best results, event images should be approximately 2:1 aspect ratio</em></p>
```

Comment:
I'm aware some (many?) of our users may not know what a 2:1 aspect ratio means but I'm not sure how best to phrase this hint otherwise.